app.tests.ts

```diff
@@ -52,23 +52,6 @@ initial_height = 595012
 describe("Command-line arguments handling", () => {
   const initialArgv = process.argv;
 
-  const helpMessage = `
```

Comment:
At first glance, removing the test message from here (and making the other one public) makes the tests seem less trustworthy. If the app's help message changes, will the tests break?